components/style/themes/default.less

```diff
@@ -394,6 +394,15 @@
 @card-actions-background: @background-color-light;
 @card-shadow: 0 2px 8px rgba(0, 0, 0, .09);
 
+// Comment
+// ---
+@comment-padding-base: 16px 0;
+@comment-indent: 44px;
+@comment-author-name-color: #8c8c8c;
+@comment-author-time-color: #ccc;
+@comment-action-color: #8c8c8c;
```

Comment:
`@comment-action-color: @text-color-secondary;`
Use the variable which is defined upper.